packages/shared/src/UI/modals/ApplicationBoardModal/ApplicationBoardForm.tsx

```diff
@@ -47,21 +69,13 @@ export function ApplicationBoardForm(props: ApplicationBoardFormProps) {
         <>
             {openSettings ? (
                 <TabContext value={currentTab}>
-                    <Stack
-                        sx={{
-                            display: 'flex',
-                            justifyContent: 'flex-end',
-                            padding: 1,
-                            flexDirection: 'column',
-                            background:
-                                'linear-gradient(rgba(255, 255, 255, 0) 0%, rgba(255, 255, 255, 0.9) 100%), linear-gradient(90deg, rgba(98, 152, 234, 0.2) 1.03%, rgba(98, 152, 234, 0.2) 1.04%, rgba(98, 126, 234, 0.2) 100%);',
-                        }}>
+                    <Stack className={classes.title}>
                         <div style={{ display: 'flex', flex: 1, justifyContent: 'start', padding: 8 }}>
                             <IconButton
                                 size="small"
                                 sx={{ margin: '-5px' }}
                                 onClick={() => setOpenSettings((openSettings) => !openSettings)}>
-                                <Icons.ArrowBack size={24} />
+                                <ArrowBackRoundedIcon />
```

Comment:
<img width="406" alt="image" src="https://github.com/DimensionDev/Maskbook/assets/1141198/c3ca33a6-ac15-45ee-8ee5-598769459c75">

What about `ComeBack`?
We better use our own icons, if no one matches, we could as designer to add.

Reply:
It's okay for a demo.